.travis.yml

```diff
@@ -9,10 +9,8 @@ osx_image: xcode6.4
 env:
   matrix:
     
-    - CONDA_NPY=111  CONDA_PY=35
-    - CONDA_NPY=112  CONDA_PY=35
-    - CONDA_NPY=111  CONDA_PY=36
-    - CONDA_NPY=112  CONDA_PY=36
+    - CONDA_PY=35
+    - CONDA_PY=36
```

Comment:
This is a new thing. We are dropping the `numpy x.x` syntax and building with the oldest `numpy` that is forwards compatible. It simplifies the build matrix and help us save some CIs cycles.